stories/SQFormDialogStepper.stories.tsx

```diff
@@ -115,11 +115,16 @@ export const WithValidation = (
           label="Personal Data"
           validationSchema={Yup.object({
             firstName: Yup.string().required(),
-            lastName: Yup.string().required(),
+            middleName: Yup.string(),
+            lastName: Yup.string()
+              .required('Required')
```

Comment:
`'Required'` is the default so this isn't really needed, like in `firstName` above.